src/exml_query.erl

```diff
@@ -34,12 +34,39 @@
 
 -export_type([path/0]).
 
-%% @doc gets the element/attr/cdata contained in the leftmost path
+%% @doc Gets the element/attr/cdata contained in the leftmost path
+%% Describe a path in the `xml' tree by pattern-matching its structure.
```

Comment:
This could be reworded, the sentence looks strange - the path is found, not not "described" by pattern-matching.